pom.xml

```diff
@@ -162,7 +162,7 @@
                         </goals>
                         <phase>package</phase>
                         <configuration>
-                            <artifact>org.apache.sling:org.apache.sling.feature.launcher:1.1.6</artifact>
+                            <artifact>org.apache.sling:org.apache.sling.feature.launcher:1.2.0</artifact>
```

Comment:
@enapps-enorman - after this change launching the Starter does not work anymore:

```
$ java -jar target/dependency/org.apache.sling.feature.launcher.jar
Error: Unable to initialize main class org.apache.sling.feature.launcher.impl.Main
Caused by: java.lang.NoClassDefFoundError: org/apache/commons/cli/ParseException
```

I'll revert just the feature launcher update while we sort this out.

Reply:
https://github.com/apache/sling-org-apache-sling-starter/pull/52